pkg/controller/appinst_api_test.go

```diff
@@ -922,19 +922,19 @@ func testSingleKubernetesCloudlet(t *testing.T, ctx context.Context, apis *AllAp
 	}, {
 		"MT specified correct cluster",
 		0, &zoneMT, mtOrg, mtClust, notDedicatedIp,
-		"shared.singlek8smt-unittest.local." + appDnsRoot, PASS,
+		"pillimogo1-atlanticinc.singlek8smt-unittest.local." + appDnsRoot, PASS,
```

Comment:
These changed test cases are all HTTP-based apps? It's hard to cross-reference with which appinst this is. I looked at `AppData()[0]`, but this doesn't look like it should get a dedicated fqdn....

Reply:
Yeah, they all use AppData()[0] which has an http accessport.